stl/inc/xstring

```diff
@@ -217,6 +224,9 @@ public:
     using pos_type   = streampos;
     using off_type   = streamoff;
     using state_type = mbstate_t;
+#if _HAS_CXX20
+    using comparison_category = strong_ordering;
+#endif // _HAS_CXX20

```

Comment:
Do we need to repeat this here given that `_Char_traits` already defines it? (Also on line 368.)